src/confirmation-validator.ts

```diff
@@ -11,7 +12,24 @@ export class ConfirmationValidator extends Validator {
 
   // TODO: POST /connection_requests
 
-  // TODO: POST /primary_validator_updated
+  /**
+   * Sends a notification to the bank that a primary validator has left the network.
+   * @param ipAddress the ip address of the primary validator that is leaving
+   * @param port the port that the primary validator is on
+   * @param protocol the protocol of the primary validator
+   * @param serverAccount the account that the current `ConfirmationValidator` is connected to
+   */
+  async sendPrimaryValidatorUpdatedPing(
+    ipAddress: string,
+    port: string,
+    protocol: string,
+    serverAccount: Account
+  ): Promise<any> {
```

Comment:
Please don't include the return type as that is implicit.  Sorry that my prompt may have inclined you to put it there.

Reply:
No problem. Fixed that